MaxText/tests/llama_test.py

```diff
@@ -0,0 +1,103 @@
+"""
+ Copyright 2023 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+
+""" Tests for Llama """
+import jax
+import unittest
+import jax.numpy as jnp
+from typing import Tuple
+import layers
+import numpy as np
+
+jax.config.update('jax_platform_name', 'cpu')
```

Comment:
Can you remove this and let your tests run on TPU. This config is global and all the other unit tests in MaxText assume they are running on TPU.

Reply:
As discussed, we have increased the rtol for the llama_test to 1e-02